src/common/dcsvg.cpp

```diff
@@ -46,35 +46,40 @@ namespace
 // C locale (i.e. always using "." for the decimal separator) and with the
 // fixed precision (which is 2 for some unknown reason but this is what it was
 // in this code originally).
-inline wxString NumStr(double f)
+inline wxString NumStr(double const& f)
 {
-    return wxString::FromCDouble(f, 2);
+    if ( f == 0 )
+        return wxS("0.00");
+    else
+        return wxString::FromCDouble(f, 2);
 }
 
 // Return the colour representation as HTML-like "#rrggbb" string and also
 // returns its alpha as opacity number in 0..1 range.
-wxString Col2SVG(wxColour c, float *opacity)
+wxString Col2SVG(wxColour c, float *opacity = NULL)
 {
     if ( c.Alpha() != wxALPHA_OPAQUE )
     {
-        *opacity = c.Alpha() / 255.0f;
+        if ( opacity )
+            *opacity = c.Alpha() / 255.0f;
 
         // Remove the alpha before using GetAsString(wxC2S_HTML_SYNTAX) as it
         // doesn't support colours with alpha channel.
         c = wxColour(c.GetRGB());
     }
     else // No alpha.
     {
-        *opacity = 1.;
+        if ( opacity )
+            *opacity = 1.;
     }
 
     return c.GetAsString(wxC2S_HTML_SYNTAX);
 }
 
-wxString wxPenString(wxColour c, int style = wxPENSTYLE_SOLID)
+wxString wxStrokeString(wxColour const& c, int style = wxPENSTYLE_SOLID)
```

Comment:
As long as we're renaming them, maybe we should remove the `wx` prefix from these (local, defined in an anon namespace) functions too? It's not like the prefix is used consistently in this code anyhow.